recipe/meta.yaml

```diff
@@ -15,6 +15,8 @@ source:
 
 build:
   number: 0
+  # the package absl is not available in s390x
+  skip: True  # [py<36 or s390x]
   noarch: python
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
```

Comment:
In host use `setuptools >=24.2`, see https://github.com/abseil/abseil-py/blob/0f86230d77cd90ea1236f0cbee4df9a9b290ef66/setup.py#L42